### PR TITLE
Fixed the duplicated fallback-error(s) and created a spec test for it.

### PR DESCRIPTION
**Pull request by AngryBerryMS**
Fixed the duplicated fallback-error(s) and created a spec test for it.

[Bug 273101](https://dev.azure.com/ceapex/Engineering/_workitems/edit/273101)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/6352)

### CODE REVIEW

File: docs/specs/localization.yml [hunk 1]
```diff
@@ -792,3 +792,37 @@ repos:
...
+repos:
+  https://github.com/MS/test:
+    - files:
+        docfx.yml:
+        docs/toc.md: |
+          # a 
+          ### b
+  https://docs.com/error-from-source-legacy.zh-cn:
+    - files:
+        .openpublishing.publish.config.json: |
```
Comment (yufeih) [inline docs/specs/localization.yml:808]: This file could be removed.

File: docs/specs/localization.yml [hunk 2]
```diff
@@ -792,3 +792,37 @@ repos:
 outputs:
   docs/a.json: |
     { "conceptual":"<p><a href=\"b\">link in b</a></p>\n<p><a href=\"a\">link in a</a></p>\n"}
+---
+# Remove duplicated fallback-error
+DryRun: true
```
Comment (yufeih) [inline docs/specs/localization.yml:797]: remove?
```suggestion
~~DryRun: true~~
```

Comment (AngryBerryMS) [reply docs/specs/localization.yml]: removed

File: docs/specs/localization.yml (hunk 1 above)
Comment (AngryBerryMS) [reply docs/specs/localization.yml:807]: ??

Comment (AngryBerryMS) [reply docs/specs/localization.yml]: Removed.

File: docs/specs/localization.yml [hunk 3]
```diff
@@ -792,3 +792,19 @@ repos:
...
   docs/a.json: |
     { "conceptual":"<p><a href=\"b\">link in b</a></p>\n<p><a href=\"a\">link in a</a></p>\n"}
+---
+# Remove duplicated fallback-error
+locale: zh-cn
+repos:
+  https://github.com/error-from-fallback#live:
+    - files:
+        docfx.yml:
+        docs/toc.md: |
```
Comment (live1206) [inline docs/specs/localization.yml:802]: make this `TOC.md` to pass linux test

**Comment by live1206** [comment]
> 🚗🌳💥🤕🚑[dryrun.azure-docs-pr](https://github.com/MicrosoftDocs/azure-docs-pr)
> ```diff
> System.InvalidOperationException: '"git" *** fetch origin --progress --prune dryrun.azure-docs-pr' failed in directory 'D:/docfx.TestData\regression-test.dryrun.azure-docs-pr' with exit code 128, message: 
>  fatal: unable to access 'https://dev.azure.com/ceapex/Engineering/_git/docfx.TestData/': The requested URL returned error: 503
> 
>    at Microsoft.Docs.Build.RegressionTest.Exec(String fileName, String arguments, String stdin, String cwd, Boolean ignoreError, Boolean redirectStandardError, Int32[] allowExitCodes, String[] secrets) in D:\a\1\s\test\docfx.RegressionTest\RegressionTest.cs:line 323
>    at Microsoft.Docs.Build.RegressionTest.EnsureTestData(Options opts, String workingFolder) in D:\a\1\s\test\docfx.RegressionTest\RegressionTest.cs:line 167
>    at Microsoft.Docs.Build.RegressionTest.Run(Options opts) in D:\a\1\s\test\docfx.RegressionTest\RegressionTest.cs:line 56
> ```

@mingwli

**Pull request merged**